.github/workflows/node.js.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
```

Comment:
Let's delete these comments, if you dont mind

Reply:
What's the rationale for removing the comments?

Reply:
Just thought that these were comments brought across from a template. Happy to keep them if we think we should 😄 
Was a nitpick from me really

Reply:
It was left over from the template. I guess some sort of comment could be helpful though 🤔

Reply:
I've added a comment that better captures the intent of the workflow

Reply:
Amazing! Yeah sorry if that was vague feedback on my part!